test/helpers/contextUtils.test.js

```diff
@@ -27,7 +27,42 @@ describe('getPatientFromContext', () => {
   });
 });
 
-describe('getConditionFromContext', () => {
+describe('getConditionsFromContext', () => {
+  const conditionResource = {
+    resourceType: 'Condition',
+    id: 'mCODEConditionExample01',
+  };
+  const conditionResource2 = {
+    resourceType: 'Condition',
+    id: 'mCODEConditionExample02',
+  };
+  const conditionContext = {
+    resourceType: 'Bundle',
+    type: 'collection',
+    entry: [
+      {
+        fullUrl: 'context-url-1',
+        resource: conditionResource,
+      },
+      {
+        fullUrl: 'context-url-2',
+        resource: conditionResource2,
+      },
+    ],
+  };
+  test('Should return Condition resource in context', () => {
+    const conditions = getConditionsFromContext(conditionContext);
+    expect(conditions).toContain(conditionResource);
+    expect(conditions).toContain(conditionResource2);
+  });
+
+  test('Should throw an error if there is no patient in context', () => {
```

Comment:
Really small suggestion
```suggestion
  test('Should throw an error if there is no Condition resource in context', () => {
```